### PR TITLE
[1LP][RFR][NOTEST] Added new manual test case

### PR DESCRIPTION
**Pull request by ganeshhubale**
- Added new manual test cases for ```auotmate``` FA
- BZ: [1684417](https://bugzilla.redhat.com/show_bug.cgi?id=1684417) is closed as not a bug hence removed related test case.

### CODE REVIEW

File: cfme/tests/automate/test_automate_manual.py [hunk 1]
```diff
@@ -1227,3 +1147,77 @@ def test_miq_stop_abort_with_state_machines():
...
+def test_user_requester_for_lifecycle_provision():
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: automate
+        testSteps:
+            1. Tested a series of service provisions and default lifecycle provisions with different
```
Comment (lina-nikiforova) [inline cfme/tests/automate/test_automate_manual.py]: I'd suggest it to put it in other words, this Jay from BZ is a bit confusing :)

File: cfme/tests/automate/test_automate_manual.py [hunk 2]
```diff
@@ -1227,3 +1147,77 @@ def test_miq_stop_abort_with_state_machines():
...
+               users (Jay only did default, usera only did generic service provision)
+            2. Added logging to the validate_request method for both Service and Infrastructure
+               namespaces
+            3. Viewed the automation.log
+        expectedResults:
+            1.
+            2.
+            3. Request user should be correct
```
Comment (lina-nikiforova) [inline cfme/tests/automate/test_automate_manual.py]: Also, it's better to define what it means "correct". Something like `The same user is shown in the automation.log as in the request`

File: cfme/tests/automate/test_automate_manual.py (hunk 1 above)
Comment (ganeshhubale) [reply cfme/tests/automate/test_automate_manual.py]: Agree! I have added more descriptionand remover Jay user.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**